src/main/java/study/SimpleCalculator.java

```diff
@@ -0,0 +1,52 @@
+package study;
+
+public class SimpleCalculator {
+
+	private final Equation equation;
+
+	private static final String NO_DIVIDE_BY_ZERO = "0으로 나눌 수 없습니다.";
+
+	public SimpleCalculator(Equation equation) {
+		this.equation = equation;
+	}
+
+	public int cal(String symbol, Integer num1, Integer num2) {
```

Comment:
메서드나 변수명에 약어를 사용하는 것은 지양해주세요.

약어는 별도의 규약으로 정해지지 않으면 소스코드의 가독성을 어렵게 만듭니다 :)

```suggestion
	public int calculate(String symbol, Integer num1, Integer num2) {
```